### DISCUSSION TITLE
MS SQL Query Builder Views

### DISCUSSION THREAD

**Discussion by stepanurban** [grafana/grafana#57598] [Data Sources]
Would it be possible to list in MS SQL Query Builder (in Tables drop-down menu) also database views? We have a lot of views in our database with human readable names and we see only encoded table names which are not understandable for us.

By view I mean
```
CREATE VIEW machine AS
SELECT
[M3255322] AS [timestamp],
[SM432334] AS [pressure_01],
[M3f332284] AS [temperature_01]
FROM raw_data_table
```
and this view is not listed in this Table drop-down menu in MS SQL Query Builder

**Comment by yosiasz** [comment]
they might be using the object type of U or USER_TABLE

```
type	type_desc
U 	USER_TABLE
V 	VIEW

select type,type_desc from sys.objects where name in ('vw_turquoise', 'tblUser')
```

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
